apps/website/src/pages/docs/button.mdx

```diff
@@ -72,10 +113,31 @@ Pressing this button will always open a menu. Upon selecting an option from the
 
 ## Ideas button
 
+The Ideas button is unique and should only be used under certain circumstances. You will need to perform an audit of your application to ensure that the feedback button does not cover / obscure any tools.
+
+If the button does obscure anything important, we suggest adding empty space at the bottom of the page so the user can scroll far enough down that the button doesn’t overlap the underlying tools.
+
 <LiveExample src='IdeasButton.main.tsx'>
   <AllExamples.IdeasButtonMainExample client:load />
 </LiveExample>
 
 ### Props
 
 <PropsTable path={frontmatter.propsPathIdeasButton} />
+
+## Usage
+
+### Style types
+
+- Use default button just about everywhere.
+- One high visibility button per section (e.g. in a modal dialog, or above the table).
+- One CTA button on a page at a time and should always appear within a dedicated page footer.
```

Comment:
We define CTA above, but if someone scrolled past it and missed it, CTA might be confusing.

The old site used [`<abbr>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr) on acronyms.  What do you makes the most sense?  We could consider this in a separate PR just to get this merged quicker.

Reply:
I would just put "(blue)" and "(green)" in parentheses.

`<abbr>` has known accessibility issues.

Reply:
and of course, we can just spell "call-to-action" too.

In addition to that, we can link back to that the sections above.

Reply:
I'd prefer to either link to the CTA section, completely spell out call-to-action, or both (no preference between these).  I would prefer to not say blue or green.

Reply:
Using call-to-action, still have CTA definition because we use that in props.